soc/nxp/imx/imx9/imx943/Kconfig.defconfig.mimx94398.m33

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMX94398_M33
+
+DT_CHOSEN_Z_FLASH := zephyr,flash
+
+config FLASH_SIZE
+	default $(dt_chosen_reg_size_int,$(DT_CHOSEN_Z_FLASH),0,K)
+
+config FLASH_BASE_ADDRESS
+	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_FLASH))
+
+config MCUX_CORE_SUFFIX
+	default "_cm33_core1" if SOC_MIMX94398_M33
+
+config NUM_IRQS
+	default 405
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 333000000
```

Comment:
Currently this need to be from dts property, refer to:
https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/gd/gd32/gd32e50x/Kconfig.defconfig.gd32e507#L7

Reply:
Thanks. Will use that.